ipld/get.go

```diff
@@ -0,0 +1,16 @@
+package ipld
+
+// import (
+// 	"context"
+//
+// 	"github.com/ipfs/go-blockservice"
+// 	"github.com/ipfs/go-cid"
+// )
+//
+// type ShareGetter struct {
+// 	getter blockservice.BlockGetter
+// }
+//
+// func (sg *ShareGetter) GetShares(ctx context.Context, root cid.Cid, len int) ([]Share, error) {
+// 	sg.getter.GetBlocks()
+// }
```

Comment:
Why commented?

Reply:
That's a rebase artifact. Pushing without this